pkg/virt-controller/services/template_test.go

```diff
@@ -513,10 +516,17 @@ var _ = Describe("Template", func() {
 				Expect(err).ToNot(HaveOccurred())
 				for _, c := range pod.Spec.Containers {
 					if c.Name != "compute" {
-						Expect(c.SecurityContext.SELinuxOptions.Level).To(Equal("s0"))
+						if enableWorkaround {
+							Expect(c.SecurityContext.SELinuxOptions.Level).To(Equal("s0"), "failed on "+c.Name)
+						} else {
+							Expect(c.SecurityContext.SELinuxOptions.Level).To(BeEmpty(), "failed on "+c.Name)
+						}
 					}
 				}
-			})
+			},
+				Entry(`"" on all virt-launcher containers`, false),
```

Comment:
I don't understand this entry's title.

Reply:
That's because I forgot to change the description on the DescribeTable, nice catch, fixing now.

Reply:
Done. Makes more sense now?

Reply:
Yes thanks!